randovania/games/dread/logic_database/Cataris.txt

```diff
@@ -481,8 +482,14 @@ Extra - asset_id: collision_camera_009
               # Energy Requirements
               Combat (Intermediate) or Normal Damage ≥ 699
               Combat (Beginner) and Normal Damage ≥ 349
-          # Methods to dodge laser
-          Flash Shift or Combat (Intermediate) or Use Spin Boost
+          Any of the following:
+              # Methods to dodge laser
+              Flash Shift or Combat (Intermediate) or Use Spin Boost
+              Combat (Intermediate) and Normal Damage ≥ 250
```

Comment:
Should take the combat out of the first line, else it will not use the damage requirement.